infrastructure/expected-fail/precondition-in-promise.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Precondition in promise</title>
```

Comment:
This test results in a failing subtest and I'd like to have `PRECONDITION_FAILED` as a test-level status.

However, I'd like to let that block on https://github.com/web-platform-tests/wpt/pull/20036 because as long as this remains ...

https://github.com/web-platform-tests/wpt/blob/0f82824745b6b43fc9c442ce0c7fc04e07cad3e7/resources/testharness.js#L3670

... then that `tests.set_file_is_test()` will still result in a bogus subtest. I'd prefer to make that change when it turns a harness `ERROR` into a harness `PRECONDITION_FAILED`.

@jugglinmike FYI.

Reply:
https://github.com/web-platform-tests/wpt/commit/783a606874219de16370a7f341727ccd1b24a470 is (roughly) the follow-up that I'd do after https://github.com/web-platform-tests/wpt/pull/20036 is resolved.

Reply:
https://github.com/web-platform-tests/wpt/pull/20036 is merged now so I'll do it all together.